app/src/androidTest/java/org/fossasia/openevent/APITest.java

```diff
@@ -36,4 +38,25 @@ public void failure(RetrofitError error) {
         });
         latch.await();
     }
+
```

Comment:
@leto It does pass. The error was in the first test, so I commented this but forgot to uncomment it after the error was fixed. I'll do it in my next commit.